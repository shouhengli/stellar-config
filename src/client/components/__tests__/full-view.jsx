const React = require('react');
const FullView = require('../full-view.jsx');
const renderer = require('react-test-renderer');

describe('component full-view', () => {
  test('wraps its children', () => {
    const component = renderer.create(<FullView>content</FullView>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
