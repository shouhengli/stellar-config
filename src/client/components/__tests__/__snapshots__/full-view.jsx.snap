// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component full-view wraps its children 1`] = `
<div
  className="view"
>
  content
</div>
`;
